app/src/main/java/com/babylon/wallet/android/presentation/account/history/HistoryViewModel.kt

```diff
@@ -118,11 +118,17 @@ class HistoryViewModel @Inject constructor(
         }
         _state.update { state ->
             state.copy(
-                timeFilterItems = result.mapIndexed { index, monthFilter ->
+                timeFilterItems = result.map { monthFilter ->
                     Selectable(monthFilter)
                 }.toPersistentList()
             )
         }
+        // if filters are computed after history loaded, we need to highlight current month
+        if (_state.value.content is State.Content.Loaded) {
+            _state.value.firstVisibleIndex?.let { index ->
+                _state.value.historyItems?.getOrNull(index)?.dateTime?.let { selectDate(it) }
+            } ?: _state.value.historyItems?.firstOrNull()?.dateTime?.let { selectDate(it) }
+        }
```

Comment:
Why is this part of this PR? Is there any problem with the branch?

Reply:
this is fix for a bug Umair reported but didn't create ticket for. I load history and months strip in paralelel, in case history load first, proper month was not selected